src/CRM/CommunityMessagesBundle/Resources/views/Default/tips.html.twig

```diff
@@ -1,44 +1,11 @@
-<style type="text/css">
-  div.civicrm-community-messages {
-    border: 2px solid #7dc857;
-    padding: 10px;
-    position: relative;
-  }
-  div.civicrm-community-messages .crm-collapsible .collapsible-title {
-    font-weight: normal;
-    color: #056085;
-  }
-  div.civicrm-community-messages .crm-collapsible .collapsible-title:hover {
-    color: #003676;
-  }
-  div.civicrm-community-messages a.civicrm-community-message-dismiss {
-    position: absolute;
-    opacity: .6;
-    top: 2px;
-    right: 2px;
-  }
-  div.civicrm-community-messages a.civicrm-community-message-dismiss:hover {
-    opacity: 1;
-  }
-  div.civicrm-community-messages .collapsed .collapsible-title div {
-    white-space: nowrap;
-    overflow: hidden;
-    text-overflow: ellipsis;
-  }
-  div.civicrm-community-messages .collapsed .collapsible-title div:after {
-    content: " \2026";
```

Comment:
I'm happy to see all this cruft go... but I was fond of this little touch. I thought adding an ellipsis to the end of the closed title would encourage people to open it.

Reply:
Ah I missed that. 

I wonder if we could add a utility class to civicrm.css for details/summary that adds an ellipsis when a classname is used on the summar. E.g. something like:

```
.crm-container details > summary.crm-accordion-tbc:after {
    content: '…';
}
.crm-container details[open] > summary.crm-accordion-tbc::after {
  content: '';
}
```

Reply:
If possible to avoid depending on civicrm-core, we could merge this sooner than later.

Could the ellipsis be added in PHP? i.e. it's always added to the title?

Reply:
If it's always there whether the alert is expanded or closed - then it could easily be added to the twig?